contrib/mpd/template.py

```diff
@@ -0,0 +1,73 @@
+pkgname = "mpd"
+pkgver = "0.23.15"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Ddocumentation=enabled",
+    "-Dhtml_manual=true",
+    "-Dmanpages=true",
+    "-Dipv6=enabled"
+]
+hostmakedepends = ["pkgconf", "meson", "python-sphinx"]
+# MikMod causes a crash
+makedepends = [
+    "alsa-lib-devel",
+    "audiofile-devel",
+    "avahi-devel",
+    "boost-devel",
+    "bzip2-devel",
+    "chromaprint-devel",
+    "fdk-aac-devel",
+    "ffmpeg-devel",
+    "flac-devel",
+    "fluidsynth-devel",
+    "fmt-devel",
+    "lame-devel",
+    "libcdio-paranoia-devel",
+    "libcurl-devel",
+    "libexpat-devel",
+    "libgme-devel",
+    "libid3tag-devel",
+    "libmms-devel",
+    "libmodplug-devel",
+    "libnfs-devel",
+    "libpulse-devel",
+    "libsamplerate-devel",
+    "libsndfile-devel",
+    "liburing-devel",
+    "libvorbis-devel",
+    "mpg123-devel",
+    "openal-soft-devel",
+    "opus-devel",
+    "pcre2-devel",
+    "pipewire-devel",
+    "pipewire-jack-devel",
+    "soxr-devel",
+    "sqlite-devel",
+    "twolame-devel",
+    "wavpack-devel",
+    "yajl-devel",
+    "zlib-devel",
+]
+pkgdesc = "Music player daemon"
+maintainer = "nullobsi <nullobsi@unix.dog>"
+license = "GPL-2.0-or-later AND BSD-2-Clause"
+url = "https://www.musicpd.org"
+source = f"https://www.musicpd.org/download/mpd/{pkgver[:-3]}/mpd-{pkgver}.tar.xz"
+sha256 = "550132239ad1acf82ccf8905b56cc13dc2c81a4489b96fba7731b3049907661a"
+
+make_dir = "built"
```

Comment:
this goes near the top (i assume it fails since build/ already exists, but then add a comment for why)